DepartmentDetails/src/com/fsd/group4/departments/HrDepartment.java

```diff
@@ -0,0 +1,34 @@
+package com.fsd.group4.departments;
+
+import com.fsd.group4.departmentinterfaces.IDoActivity;
+
+public class HrDepartment extends SuperDepartment implements IDoActivity{
+
+  /*
+   * departmentName will return “ Hr Department “
+   * getTodaysWork will return “ Fill today’s timesheet and mark your attendance”
+   * getWorkDeadline will return “ Complete by EOD “
+   * doActivity “team Lunch”
+   * 
+   */
+  @Override
+  public String departmentName() {
+    return "Hr Department ";
+  }
+
+  @Override
+  public String getTodaysWork() {
+    return "Fill today's timesheet and mark your attendance";
```

Comment:
As per assignment return string should be " Fill today’s timesheet and mark your attendance"